src/MagnumPlugins/BasisImageConverter/BasisImageConverter.h

```diff
@@ -218,8 +229,14 @@ class MAGNUM_BASISIMAGECONVERTER_EXPORT BasisImageConverter: public AbstractImag
 
     private:
         MAGNUM_BASISIMAGECONVERTER_LOCAL ImageConverterFeatures doFeatures() const override;
+
         MAGNUM_BASISIMAGECONVERTER_LOCAL Containers::Array<char> doConvertToData(Containers::ArrayView<const ImageView2D> imageLevels) override;
+        MAGNUM_BASISIMAGECONVERTER_LOCAL Containers::Array<char> doConvertToData(Containers::ArrayView<const ImageView3D> imageLevels) override;
+
+        template<UnsignedInt dimensions> bool convertLevelsToFile(const Containers::ArrayView<const BasicImageView<dimensions>> imageLevels, const Containers::StringView filename);
```

Comment:
Last thing: 

```suggestion
        MAGNUM_BASISIMAGECONVERTER_LOCAL template<UnsignedInt dimensions> bool convertLevelsToFile(const Containers::ArrayView<const BasicImageView<dimensions>> imageLevels, const Containers::StringView filename);
```

Reply:
Done, but GCC doesn't seem to like it before the `template<>`, only works after it or after the return value type.

Reply:
Ah, yes, after the template, sorry. Attribute placement rules are crazy.